Configurations/10-main.conf

```diff
@@ -865,7 +862,7 @@ sub vms_info {
         dso_scheme       => "dlfcn",
         shared_target    => "linux-shared",
         shared_cflag     => "--pic",
-        shared_ldflag    => add("-z --sysv --shared"),
```

Comment:
I'd like to insist on keeping this. Note two dashes, i.e. it's not something that is common. However, this config is not actually fully operational [yet], so if one has to actually do something with it *now*, it probably would be more appropriate to say 'shared_target => "unknown" or remove 'shared_target' altogether?

Reply:
Makes sense, I was apprehensive of that one.  I'll remove `shared_target`

Reply:
I decided the other way, to leave it as it was (i.e. --shared restored).  I don't know the platform, so less sudden surprises, right?

Reply:
I know everything about it, and I find it hard to imagine that anybody else knows something. So either way is fine... I mean as long as --shared stays...